modules/bookAddRemove.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable */
+const titleInput = document.querySelector('.title-input');
+const authorInput = document.querySelector('.author-input');
+const addBtn = document.querySelector('.add-input');
+let bookArr = [];
+
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
+
+  add() {
+    const book = document.createElement('div');
+    book.className = 'book';
+    book.setAttribute('id', this.id);
+    const title = document.createElement('span');
+    title.className = 'title';
+    title.textContent = this.title;
+    book.appendChild(title);
+    const author = document.createElement('span');
+    author.className = 'author';
+    author.textContent = this.author;
+    book.appendChild(author);
+    const removeBtn = document.createElement('button');
+    removeBtn.className = 'remove-btn';
+    removeBtn.textContent = 'Remove';
+    removeBtn.addEventListener('click', (e) => {
+      Book.remove(e);
+    });
+    book.appendChild(removeBtn);
+    document.querySelector('.books-container').appendChild(book);
+  }
+
+  static remove(e) {
+    bookArr = bookArr.filter((each) => e.target.parentElement.id !== each.id);
+    localStorage.setItem('books', JSON.stringify(bookArr));
+    e.target.parentElement.remove();
+  }
+}
```

Comment:
- [ ] Kindly make sure that a user can delete any book from the app and it persists in the local storage. Right now, after deleting and refreshing the page, these two books still appear. Kindly rectify that 👍 
- Your app:
![Screenshot 2022-07-05 141302](https://user-images.githubusercontent.com/85113567/177336385-feb44b61-7680-4ce5-8c8e-27794e12e347.png)

Reply:
I have addressed this situation as advised. Thanks